Nos/Models/CoreData/Relay+CoreDataClass.swift

```diff
@@ -143,4 +143,11 @@ public class Relay: NosManagedObject {
     var hasMetadata: Bool {
         metadataFetchedAt != nil
     }
+    
+    var displayAddress: String? {
+        if let address {
+            return address.replacingOccurrences(of: "wss://", with: "")
+        }
+        return nil
```

Comment:
Maybe this can all fit in one line?

```suggestion
        address?.replacingOccurrences(of: "wss://", with: "")
```

Or, wait: can we just use relay.host the way `RelayPicker` does? Maybe we don't need a `displayAddress` at all.